src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figures implements Drawable {
```

Comment:
Нужно еще создать интерфейс AreaCalculator с методом getArea() и импелментировать его классом Figure.
```suggestion
public abstract class Figure implements Drawable, AreaCalculator  {
```